.gitignore

```diff
@@ -48,11 +48,13 @@ android/keystores/debug.keystore
 .flowconfig
 .gitattributes
 .watchmanconfig
-android/app/src/main/res/mipmap-hdpi/ic_launcher_round.png
-android/app/src/main/res/mipmap-mdpi/ic_launcher_round.png
-android/app/src/main/res/mipmap-xhdpi/ic_launcher_round.png
-android/app/src/main/res/mipmap-xxhdpi/ic_launcher_round.png
-android/app/src/main/res/mipmap-xxxhdpi/
+android/app/src/main/res/mipmap-hdpi/
```

Comment:
Theres actually no need to ignore these folders as they're empty.

Reply:
just added they won't do anything sitting there 🤓